emission/tests/analysisTests/TestClusterPipeline.py

```diff
@@ -1,171 +0,0 @@
-import unittest
-import emission.core.get_database as edb
-import emission.analysis.modelling.tour_model.cluster_pipeline as cp
-import emission.simulation.trip_gen as tg
-import uuid as uu
-
-class ClusterPipelineTests(unittest.TestCase):
```

Comment:
Can you comment more on why you had to remove these tests? I know you said that they weren't valid any more, but I'd like to understand why, and to see how much effort it would be to port them over.